### DISCUSSION TITLE
How to retrieve the account name set in the browser extension from a web app?

### DISCUSSION THREAD

**Discussion by icfor** [phantom/docs#218] [Help & Support] [answered]
### Summary

Is there a way to retrieve the name that the user has set to the wallet in the browser extension? For example with a RPC method using `phantom.solana.request`. Many thanks!

### Steps to Reproduce

At the top of the browser extension, I can change the name of my current wallet account. Would like to ask if it is possible to retrieve this value from a web application that has connected with the extension using `phantom.solana.connect`.

### Phantom Version

24.5.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @icfor,

Thank you for writing in! It is not possible to retrieve the user defined account name in Phantom. Please feel free to open a feature request [here.](http://phantom.canny.io)
